### ISSUE TITLE
Highlighted links and register inconsistencies

### ISSUE THREAD

**Issue by martinantonmueller** [acdh-oeaw/schnitzler-tagebuch-static#18]
There are several shortcomings of having the links to indexed entries within the edited text:
* There are about 1/3 of entries missing (see the earlier issues), that's why we reverted back to the lists that tell on which day which person is mentioned
* The works are not highlighted even though we now have a register
* It kind of destroys the text by making it less readable and more of a search for the links-view

So to sum it up I would strongly support not having links in the text and just have the register menu for each day
